README.md

```diff
@@ -1,2 +1,77 @@
+and controlled path for new features to enter their project, so that new
+contributors can anticipate the short- and long-term development roadmap of the
+project.
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Introduction](#cartographer-rfcs)
+  - [Table of Contents]
+  - [When should you write an RFC]
+  - [Motivation for RFCs]
+  - [Before creating an RFC]
+  - [What the process is]
+
+## When should you write an RFC
+[When should you write an RFC]: #when-should-you-write-an-rfc
+
+You should provide an RFC if you intent to make a *significant* change or
+contribution to one of Cartographer's repositories. A change is e.g.
+*significant* if it introduces a new or modifies an existing *concept*. The
+process is really very lightweight so we encourage you to err on the side of
+RFCs.
+
+## Motivation for RFCs
+[Motivation for RFCs]: #motivation-for-rfcs
+
+Before the introduction of RFCs reviewer's would sometimes spend time reading
+incoming PRs trying to find out what the problem is the author is actually
+solving. This could slow down reviews considerably which is unfortunate for both
+the reviewer and contributor.
+
+Sometimes a PR could sit in the review queue for a long time as it was modifying
+code that was undergoing a refactoring and merging it would make that
+refactoring harder. So the purpose of a RFC is both to vet incoming feature
+proposals and also allow other contributors to learn about on-going projects
+and coordinate better.
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+Before opening an RFC we ask you to briefly look over the list of RFCs being
+discussed or [implemented][Cartographer Projects]. If your RFC conflicts with
+another RFC please inject yourself into the discussion by either commenting on
+the RFCs PR or opening an issue about the RFC being implemented in this
+repository.
+
+## What the process is
+[What the process is]: #what-the-process-is
+
+Follow this process to get your feature merged into Cartographer.
+
+  - Fork the RFC repo [RFC repository]
+  - Copy `0000-template.md` to `text/0000-my-feature.md` (where "my-feature" is
+    descriptive. Don't assign an RFC number yet).
+  - Fill in the RFC.
+  - Submit a pull request. As a pull request the RFC will receive design
+    feedback from the larger community, and the author should be prepared to
+    revise it in response.
+  - When consensus about the RFC has been reached the corresponding PR will be
+    approved and the PR merged.
+  - The person that actually merges the PR will create a new *project* for the
+    duration of the implementation (which might span several PRs) in the
+    projects pane of the google-cartographer organization and assign an RFC
+    number.
+  - When you create implementing PRs, please add a link to the RFC in the PR
+    description.
+
+[Cartographer Projects]: https://github.com/googlecartographer/cartographer/projects
```

Comment:
Why don't we have an organization level project page? as @SirVer suggested [here](https://github.com/googlecartographer/rfcs/pull/1#discussion_r147684523)? You can enable projects for organization under project section in organization section. This [link](https://github.com/organizations/googlecartographer/settings/projects) might work

Reply:
Right, that was my idea and I had already updated the text accordingly but forgotten to do the same for the link to the projects page. Fixed now. Thanks.

Reply:
Oops.. sorry @cschuet, I just found out that [Org project board cannot be public](https://stackoverflow.com/questions/40532661/make-github-organization-projects-public)... The project board is only visible to the organization members.

There is an alternative way of making it public but I am not sure if it is good. See [this post](https://medium.com/@bmpvieira/how-to-make-your-github-organization-project-board-public-fb5a243f28e4)

Reply:
Oh thanks for pointing out I did not know about that. So how about we use the RFCs project pane instead?

Reply:
I am fine with it. I am actually fine with just using cartographer project page too.

Reply:
I think the cartographer project page is most appropriate.

Reply:
Done.